packages/web-app-unzip/vite.config.ts

```diff
@@ -2,16 +2,6 @@ import { defineConfig } from '@opencloud-eu/extension-sdk'
 
 export default defineConfig({
   name: 'web-app-unzip',
```

Comment:
Should this be `unzip`?

Reply:
It's only a name, so it doesnt matter, but yes, that would be more consistent. Slipped through apparently. Good catch, thanks!